BOOKBBANG/BOOKBBANG/Views/Phrase/AddThoughts/AddThoughts.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  AddThoughts.swift
+//  BOOKBBANG
+//
+//  Created by kyunglimkim on 8/1/24.
+//
+
+import SwiftUI
+
+struct AddThoughts: View {
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            AddThoughtsHeader()
+            
+            AddThoughtsProgressBar()
+            
+            Spacer() .frame(height: 132)
+            
+            Section {
+                ZStack(alignment: .trailing) {
+                    Rectangle()
+                        .foregroundStyle(.greenMain40)
+                        .frame(height: 14)
+                        .padding(.bottom, -10)
+                    
+                    Text("\"당신이 누군가를 죽였다\"")
+                        .foregroundStyle(.typo100)
+                        .font(.system(size: 24, weight: .bold))
+                        .padding(.bottom, 3)
+                }
+                .fixedSize()
+                
+                
+                Text("이 빵에 어떤 생각을 담아볼까요?")
+                    .foregroundStyle(.typo100)
+                    .font(.system(size: 24, weight: .bold))
+                    .padding(.bottom, 10)
+                Text("나중에 다시 문구를 저장한 이유도 확인하기 좋아요")
```

Comment:
```suggestion
                Text("나중에 확인할 때 저장한 이유를 쉽게 알 수 있어요")
```

이런 식으로 라이팅을 변경해보는 건 어떨까요? 뭔가 가독성이 썩 좋진 않은 느낌이네용